Add sample: Inspecting assembly contents using MetadataLoadContext (#2053)
dotnet/samples: core/assembly/MetadataLoadContext/readme.md

---
languages:
- csharp
products:
- dotnet-core
page_type: sample
name: "Inspect assembly contents using MetadataLoadContext"
urlFragment: "inspect-assembly-contents-using-metadataloadcontext"

> description: "A sample console application that shows how to use MetadataLoadContext to load an assembly for inspection purposes"

Reviewer: ```suggestion
description: "A sample console application that shows how to use MetadataLoadContext to load an assembly for inspection purposes."
```

---
# Inspect assembly contents using MetadataLoadContext

> This sample is a .NET Core console application that loads the specified assembly and prints its custom attributes and defined types. It demonstrates how to use the [System.Reflection.MetadataLoadContext](https://www.nuget.org/packages/System.Reflection.MetadataLoadContext) API to load an assembly for inspection purposes.

Reviewer: ```suggestion
This sample is a .NET Core console application that loads a specified assembly and prints its custom attributes and defined types. It demonstrates how to use the [System.Reflection.MetadataLoadContext](https://www.nuget.org/packages/System.Reflection.MetadataLoadContext) API to load an assembly for inspection purposes.
```

## Build and run the sample

> After downloading the source, open the console in the sample directory and use following commands:

Reviewer: ```suggestion
After downloading the source, open a command prompt in the sample directory and enter the following commands:
```
